src/hipscat_cloudtests/example_module.py

```diff
@@ -0,0 +1,23 @@
+"""An example module containing simplistic functions."""
+
+
+def greetings() -> str:
+    """A friendly greeting for a future friend.
+
+    Returns
+    -------
+    str
+        A typical greeting from a software engineer.
+    """
+    return "Hello from LINCC-Frameworks!"
+
+
+def meaning() -> int:
+    """The meaning of life, the universe, and everything.
+
+    Returns
+    -------
+    int
+        The meaning of life.
+    """
+    return 42
```

Comment:
Do we need this file?

Reply:
Just for now. Created https://github.com/astronomy-commons/hipscat-cloudtests/issues/2 to force m'self to add some benchmarks, but this file keeps things compiling.